src/modules/dashboard/components/TokenMintDialog/TokenMintForm.tsx

```diff
@@ -72,6 +76,12 @@ const TokenMintForm = ({
 
   const requiredRoles: ColonyRole[] = [ColonyRole.Root];
 
+  const { isColonyReputationZero } = useColonyReputation(colony.colonyAddress);
+
+  const onlyForceAction =
+    isVotingExtensionEnabled && isColonyReputationZero && !values.forceAction;
```

Comment:
Shouldn't the force toggle also check if the person perming the forced action, actually has the required permissions ?

Or do we check that some place else ?

Reply:
Aren't those permission checked by the line below this one? 

`const inputDisabled = !userHasPermission || onlyForceAction;`

Reply:
yeah, it's done separately

Reply:
Actually, it is checked when displaying Toggle. But it made me think of something and i've added a little missing part - `colonyHasReputation` to this check:
```
const userHasPermission =
    canMintTokens || (isVotingExtensionEnabled && colonyHasReputation);
```